### PR TITLE
Update for 1.19.4

### PR DESCRIPTION
**Pull request by Konicai**
With https://github.com/GeyserMC/mappings/pull/80

- [x] items
- [x] blocks
- [x] ???

### CODE REVIEW

File: src/main/java/org/geysermc/generator/MappingsGenerator.java [hunk 1]
```diff
@@ -1055,7 +1081,8 @@ else if (trimmedIdentifier.contains("big_dripleaf")) {
             stateIdentifier = "minecraft:cobblestone_wall";
         }
 
-        if (bedrockIdentifier.startsWith("minecraft:leaves")) {
+        // todo: maybe remove cherry case when bedrock gets it
+        if (bedrockIdentifier.startsWith("minecraft:leaves") && !trimmedIdentifier.contains("cherry")) {
```
Comment (Camotoy) [inline src/main/java/org/geysermc/generator/MappingsGenerator.java:1085]: Can you put the `cherry` contains check in its own method so we can yeet it for 1.20 or whenever?

File: src/main/java/org/geysermc/generator/MappingsGenerator.java [hunk 2]
```diff
@@ -832,6 +832,32 @@ public JsonObject getRemapBlock(BlockState state, String identifier) {
             bedrockIdentifier = "minecraft:moving_block";
         } else if (trimmedIdentifier.endsWith("note_block")) {
             bedrockIdentifier = "minecraft:noteblock";
+        } else if (trimmedIdentifier.equals("minecraft:cherry_sign")) {
```
Comment (Camotoy) [inline src/main/java/org/geysermc/generator/MappingsGenerator.java:835]: Can we use a map? Should make things easier for understanding when it's time to yeet the mappings.

Comment (Konicai) [reply src/main/java/org/geysermc/generator/MappingsGenerator.java:835]: Yeah haha, I totally agree. I was getting disillusioned at this point

File: src/main/java/org/geysermc/generator/MappingsGenerator.java [hunk 3]
```diff
@@ -242,7 +302,7 @@ public void generateItems() {
             for (int i = 0; i < BuiltInRegistries.ITEM.size(); i++) {
                 Item value = BuiltInRegistries.ITEM.byId(i);
                 ResourceLocation key = BuiltInRegistries.ITEM.getKey(value);
-                if (key.getPath().endsWith("planks") && !key.getPath().contains("bamboo")) {
+                if (key.getPath().endsWith("planks")) {
```
Comment (Konicai) [inline src/main/java/org/geysermc/generator/MappingsGenerator.java:305]: I think this was a mistake from my last PR

**Pull request merged**